tests/ContentFile.py

```diff
@@ -37,7 +37,9 @@
 class ContentFile(Framework.TestCase):
     def setUp(self):
         Framework.TestCase.setUp(self)
-        self.file = self.g.get_user().get_repo("PyGithub").get_readme()
+        repo = self.g.get_user().get_repo("PyGithub")
+        self.file = repo.get_readme()
+        self.specialCharsFile = repo.get_contents("#testSpecialCharacters.txt")
```

Comment:
You only use this in the one test case, so I would export repo as self.repo and then say specialCharFile = self.repo.get_contents in the test case.